subprojects/execution/src/test/groovy/org/gradle/internal/execution/steps/StepSpec.groovy

```diff
@@ -45,6 +46,7 @@ abstract class StepSpec<C extends Context> extends Specification {
         _ * context.identity >> identity
         _ * work.displayName >> displayName
         _ * work.identify(_, _) >> identity
+        temporaryFolder.cleanup()
```

Comment:
Shouldn't this go into `cleanup()`?

Reply:
I removed all of this in: https://github.com/gradle/gradle/pull/22067